AppCenterDataStorage/AppCenterDataStorage/Internal/Util/MSDocumentUtils.m

```diff
@@ -198,4 +212,32 @@ + (BOOL)isSerializableDocument:(Class)classType {
   return class_conformsToProtocol(classType, @protocol(MSSerializableDocument));
 }
 
+/* Serialize NSDate instances to ISO8601 strings. Returns that dictionary if
+   it is serializable, nil otherwise. */
++ (NSDictionary *)tryToMakeSerializableDictionary:(NSDictionary *)dictionary {
+  NSMutableDictionary *newDictionary = [NSMutableDictionary new];
```

Comment:
@clpolet @jaelim-ms any idea if there is a faster way to get a copy of a dictionary (as mutable)?

Reply:
Maybe https://stackoverflow.com/a/17077530.